docs/general/faq/community-faqs.md

```diff
@@ -0,0 +1,349 @@
+
+Mainnet URL: ws://main.confluxrpc.org/ws
+Testnet URL: ws://test.confluxrpc.org/ws
+
+### **When `docker pull confluxchain/conflux-rust` prompts "no such file", how to solve it?**
+
+It might be a network issue. You can try setting a registry for docker:
+
+```json
+
+{
+  "registry-mirrors": [
+    "https://at8ak49f.mirror.aliyuncs.com"
+  ],
+  "experimental": false,
+  "debug": true}
+
+```
+
+### **When we use the wallet to transfer, we are asked to enter the set password, and then the transfer is successful. Why use symmetric encryption instead of a private key here?**
+
+Entering a private key is too cumbersome and hard to remember. This password is the portal (wallet) password. The private key is stored inside the portal wallet. Essentially, only the private key can authorize sending a transaction. The password is mainly for easier wallet (private key) management and to add a layer of protection. If you have very high security requirements, it is recommended to use a hardware wallet.
+
+### **What is block height?**
+
+A blockchain is a block followed by another block, continuously growing. Block height refers to the distance from this block to the genesis block.
+
+### **What is transaction signing?**
+
+Transaction signing refers to a signature obtained from a transaction through a signing algorithm using a private key. The verifier can obtain the address corresponding to the private key through this signature and transaction information, thereby proving that the transaction was indeed initiated by this account address.
+
+### **What are the measurement units in the conflux system and their conversion relationships?**
+
+The smallest unit in conflux is drip.
+
+- 1cfx = 10^18 drip
+- 1gdrip = 10^9 drip
+
+### **What does the error `estimateGasAndColletral` mean? "Cannot estimate: transaction execution failed, all gas will be charged (execution error: VmError(BadInstruction { instruction: 169 }))"**
+
+Reason: The constructor was invalid when deploying the contract; it is usually caused by forgetting to write the **`to`** address when calling the contract.
+
+### **How do developers start a node?**
+
+- Testnet node: [Link](https://github.com/Conflux-Chain/conflux-rust/releases/tag/v1.1.0-testnet)
+- Mainnet node: [Link](https://github.com/Conflux-Chain/conflux-rust/releases/tag/v1.1.0)
+- Private node: Set mode = "dev" in the configuration file.
+
+To start the node, use the command: **`./conflux --config ./xxx.toml --full 2>stderr`**.
+
+### **What should you pay attention to before sending a transaction?**
+
+Remember to set gas + strongeLimit (you can get it by executing **`estimateGasAndCollateral`**). **`strongeLimit`** is where Conflux differs from Ethereum. When using the SDK, the SDK will automatically handle it before sending the transaction.
+
+### **How many states are there in a block, and what is the order?**
+
+There are 3 states of a block: mined, executed, confirmed.
```

Comment:
Finalized state is added. A simple introduction of the finalized is suggested here or provide the link to the https://doc.confluxnetwork.org/docs/general/conflux-basics/transactions#transaction-lifecycle. (both are viable but I think the former solution would be better)